[2024-03-07T04:13:55.418+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Extraçao_dos_dados manual__2024-03-07T04:13:53.292653+00:00 [queued]>
[2024-03-07T04:13:55.435+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Extraçao_dos_dados manual__2024-03-07T04:13:53.292653+00:00 [queued]>
[2024-03-07T04:13:55.436+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-03-07T04:13:55.571+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Extraçao_dos_dados> on 2024-03-07 04:13:53.292653+00:00
[2024-03-07T04:13:55.624+0000] {standard_task_runner.py:60} INFO - Started process 5841 to run task
[2024-03-07T04:13:55.631+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Extraçao_de_dados_do_linkedin', 'Extraçao_dos_dados', 'manual__2024-03-07T04:13:53.292653+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/dag_exemplo.py', '--cfg-path', '/tmp/tmpc4fgj7fi']
[2024-03-07T04:13:55.634+0000] {standard_task_runner.py:88} INFO - Job 86: Subtask Extraçao_dos_dados
[2024-03-07T04:13:55.725+0000] {task_command.py:423} INFO - Running <TaskInstance: Extraçao_de_dados_do_linkedin.Extraçao_dos_dados manual__2024-03-07T04:13:53.292653+00:00 [running]> on host 4fca1e22de4a
[2024-03-07T04:13:55.928+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Extraçao_de_dados_do_linkedin' AIRFLOW_CTX_TASK_ID='Extraçao_dos_dados' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T04:13:53.292653+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-07T04:13:53.292653+00:00'
[2024-03-07T04:16:33.061+0000] {warnings.py:109} WARNING - /opt/***/dags/scrap/main.py:50: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 50 of the file /opt/***/dags/scrap/main.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.

  soup = BeautifulSoup(data)

[2024-03-07T04:16:41.728+0000] {logging_mixin.py:188} INFO - 0
[2024-03-07T04:16:52.670+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-07T04:16:52.683+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Extraçao_de_dados_do_linkedin, task_id=Extraçao_dos_dados, execution_date=20240307T041353, start_date=20240307T041355, end_date=20240307T041652
[2024-03-07T04:16:52.831+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T04:16:52.875+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
